### PR TITLE
update leftover status-tag usage to status template tag 

### PR DESCRIPTION
**Pull request by HimanshuGarg47**
This PR addresses this issue #10129 Part 1

_Please check the following:_

-   [x] Do the tests still pass?[^1]
-   [x] Does the code comply with the style guide?
    -   [x] Run `make lint` from the Wagtail root.
-   [ ] For Python changes: Have you added tests to cover the new/fixed behaviour?
-   [ ] For front-end changes: Did you test on all of Wagtail’s supported environments?[^2]
    -   [ ] **Please list the exact browser and operating system versions you tested**:
    -   [ ] **Please list which assistive technologies [^3] you tested**:
-   [ ] For new features: Has the documentation been updated accordingly?

**Please describe additional details for testing this change**.

[^1]: [Development Testing](https://docs.wagtail.org/en/latest/contributing/developing.html#testing)
[^2]: [Browser and device support](https://docs.wagtail.org/en/latest/contributing/developing.html#browser-and-device-support)
[^3]: [Accessibility Target](https://docs.wagtail.org/en/latest/contributing/developing.html#accessibility-targets)

### CODE REVIEW

**Comment by salty-ivy** [comment]
make sure you install the code liniting package that wagtail use, it reindents your code so you don't get those silly indentation errors as your CI failures.

additionally for part 2 as you would be dealing with changes in CSS classes and their implementations.  after the changes 
make sure you run `npm run build`.

**Comment by salty-ivy** [comment]
@HimanshuGarg47 please read the comment above the test failures are because of incorrect indentation and spacing. 
That's why I recommended you to use make lint.

**Comment by HimanshuGarg47** [comment]
> 
Make lint is giving below error
image

**Comment by salty-ivy** [comment]
You just need to install the package using `pre-commit install `

**Comment by salty-ivy** [comment]
Checkout https://docs.wagtail.org/en/stable/contributing/developing.html#automatically-lint-and-code-format-on-commits

File: wagtail/admin/templates/wagtailadmin/shared/workflow_history/detail.html [hunk 1]
```diff
@@ -78,16 +78,16 @@ <h2>{% icon "clipboard-list" classname="initial" %} {{ workflow_state.workflow.n
...
                                                 {% if task_state.finished_by %}
-                                                    {% blocktrans trimmed with action='<div class="status-tag primary">'|add:task_state.get_status_display|add:'</div>'|safe who=task_state.finished_by|user_display_name at=task_state.finished_at %}
+                                                    {% blocktrans trimmed with action='{% status task_state.get_status_display classname="primary" %}'|safe who=task_state.finished_by|user_display_name at=task_state.finished_at %}
                                                         {{ action }} by <b>{{ who }}</b> at <b>{{ at }}</b>
                                                     {% endblocktrans %}
                                                 {% else %}
-                                                    {% blocktrans trimmed with action='<div class="status-tag primary">'|add:task_state.get_status_display|add:'</div>'|safe at=task_state.finished_at %}
+                                                    {% blocktrans trimmed with action='{% status task_state.get_status_display classname="primary" %}'|safe at=task_state.finished_at %}
```
Comment (lb-) [inline wagtail/admin/templates/wagtailadmin/shared/workflow_history/detail.html:85]: Same here with the translation usage.

**Review by lb- — CHANGES_REQUESTED** [review]
Looks good somfsr except for the translation usage. Please ensure you double check usage in the bakerydemo. 

Thanks for keeping at this.

**Comment by lb-** [comment]
@HimanshuGarg47 once the fixes are applied and the CI is passing - please feel free to mark this as ready (no -draft) and add a comment it's ready for a review.

**Review by lb- — APPROVED** [review]
Tested all these scenarios and they work great, thanks so much @HimanshuGarg47 

I will rebase and get this merged in

**Pull request merged**